features/support/world.js

```diff
@@ -1,13 +1,13 @@
 'use strict';
 
 const { setWorldConstructor } = require('@cucumber/cucumber');
-const AwesomeProduct = require('../../lib');
+const AirportWeatherAPIProduct = require('../../lib');
 
 class CustomWorld {
 
     constructor() {
 
-        this.api = new AwesomeProduct();
+        this.api = new AirportWeatherAPIProduct();
```

Comment:
Let's drop the 'product' from the name. 

So, 'AirportWeatherAPIProduct' becomes 'AirportWeatherAPI'.